src/web/components/Figure.tsx

```diff
@@ -1,22 +1,30 @@
+import React from 'react';
 import { css } from 'emotion';
 
 import { from, until } from '@guardian/src-foundations/mq';
 
+type Props = {
+    role?: RoleType;
+    children: JSXElements;
+    id?: string;
+};
+
 const roleCss = {
     inline: css`
         margin-top: 12px;
         margin-bottom: 12px;
     `,
 
     supporting: css`
+        margin-top: 12px;
+        margin-bottom: 12px;
```

Comment:
I know they weren't before, but should this margins be using the `space` token from the design system?

Reply:
Yes, they should. On it.